test/TEST-61-MULTINIC/test.sh

```diff
@@ -80,8 +80,8 @@ client_test() {
         -net nic,macaddr=52:54:00:12:34:"$mac3",model=virtio \
         -netdev hubport,id=n1,hubid=1 \
         -netdev hubport,id=n2,hubid=2 \
-        -device virtio,netdev=n1,mac=52:54:00:12:34:98 \
-        -device virtio,netdev=n2,mac=52:54:00:12:34:99 \
+        -device virtio-net-pci,netdev=n1,mac=52:54:00:12:34:98 \
+        -device virtio-net-pci,netdev=n2,mac=52:54:00:12:34:99 \
```

Comment:
Can you put this change into a separate commit and reference the commit that broke it?

Reply:
The "test(MULTINIC): virtio regression" commit does not introduce a regression, but fixes one. How about adding a "fix" there?